File: sdkwork-app-sdk-typescript/src/types/payment-vo.ts
/** 支付响应 */
export interface PaymentVO {
  /** 创建时间 */
  createdAt?: string;
  /** 更新时间 */
  updatedAt?: string;
  /** 支付订单号 */
  paymentOrderId?: string;
  /** 商户订单号 */
  merchantOrderId?: string;
  /** 支付状态 */
  status?: string;
  /** 支付金额 */
  amount?: string;
  /** 支付方式 */
  paymentMethod?: string;
  /** 支付链接 */
  paymentUrl?: string;
  /** 二维码 */
  qrCode?: string;
  /** 支付ID */
  paymentId?: number;
  /** 支付流水号 */
  paymentSn?: string;
  /** 订单ID */
  orderId?: number;
  /** 订单标题 */
  subject?: string;
  /** 支付提供商 */
  paymentProvider?: string;
  /** 支付提供商名称 */
  paymentProviderName?: string;
  /** 产品类型 */
  productType?: string;
  /** 产品类型名称 */
  productTypeName?: string;
  /** 状态名称 */
  statusName?: string;
  /** 过期时间 */
  expireTime?: string;
  /** 是否需要查询 */
  needQuery?: boolean;
  /** 查询间隔 */
  queryInterval?: number;
  /** 备注 */
  remark?: string;
  /** 支付参数 */
  paymentParams?: Record<string, unknown>;
  /** 交易ID */
  transactionId?: string;
  /** 商户订单号 */
  outTradeNo?: string;
  /** 支付成功时间 */
  successTime?: string;
}

File: sdkwork-app-sdk-python/sdkwork_app_sdk/models/payment_vo.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class PaymentVO:
    """支付响应"""
    created_at: str = None
    updated_at: str = None
    payment_order_id: str = None
    merchant_order_id: str = None
    status: str = None
    amount: str = None
    payment_method: str = None
    payment_url: str = None
    qr_code: str = None
    payment_id: int = None
    payment_sn: str = None
    order_id: int = None
    subject: str = None
    payment_provider: str = None
    payment_provider_name: str = None
    product_type: str = None
    product_type_name: str = None
    status_name: str = None
    expire_time: str = None
    need_query: bool = None
    query_interval: int = None
    remark: str = None
    payment_params: Dict[str, Any] = None
    transaction_id: str = None
    out_trade_no: str = None
    success_time: str = None
